docs/sdk/migrations/datil.md

```diff
@@ -0,0 +1,176 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Migrating to Datil
+
+Previously Lit networks were powered by a pre-Bedrock Optimism L2 rollup called [Chronicle](../../network/rollup.mdx). In our effort to deliver a more performant and stable backend for Lit's infrastructure, we're launching a new network, Vesuvius, powered by [Arbitrum Orbit](https://arbitrum.io/orbit).
+
+Depending on the existing Lit network you are using, you should migrate to it's corresponding Datil network:
+
+| Currently Available | Requires Payment | Your Current Network | Network to Migrate to | Description                                                 |
+| --------------------| ---------------- | -------------------- | -------------------- | ------------------------------------------------------------ |
+|          ❌         |          ✅      | `habanero`           | `datil`               | Decentralized mainnet designed for production use cases      |
+|          ❌         |          ✅      | `manzano`            | `datil-test`          | Decentralized testnet designed for pre-production deployment |
+|          ✅         |          ❌      | `cayenne`            | `datil-dev`           | Centralized testnet designed for early-stage development     |
+
+Like their counterparts, `datil` and `datil-test` require payment for usage of the Lit network via [Capacity Credits](../capacity-credits.md); however, `datil-dev` does not.
+
+## Breaking Changes and Important Updates
+
+### Datil Resources
+
+Datil has a new [block explorer](https://vesuvius-explorer.litprotocol.com/), and Lit test tokens can be received from [this faucet](https://datil-dev-faucet.vercel.app/).
+
+### New Network, New PKPs
+
+Because existing PKPs were minted on the old Chronicle chain, migration from `cayenne` to `datil-dev` requires re-minting of all PKPs. This also means transferring ownership of assets owned by PKPs minted on Chronicle, to the newly minted ones on Chronicle Vesuvius.
+
+To reduce the friction of migrating PKPs to Datil, we've written a [migration script](https://github.com/LIT-Protocol/developer-guides-code/tree/wyatt/pkp-migration-script/pkp-migration/nodejs) that will take a list of PKP public keys, fetch their configured Auth Methods and Scopes, and mint new PKPs on a target Lit Network, setting the same Auth Methods and Scopes for each PKP.
+
+:::note
+The migration script **will not** handle migration of any assets the existing PKPs own such as tokens.
```

Comment:
maybe we can add an additional sentence of guidance here, like 'individual assets held in each PKP will need to be manually sent to the new PKP (or any other account of your choosing) by making a blockchain transaction'